netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -182,7 +193,9 @@ WriteQueue getWriteQueue() {
    */
   void returnProcessedBytes(Http2Stream stream, int bytes) {
     try {
-      decoder().flowController().consumeBytes(ctx, stream, bytes);
+      decoder().flowController().consumeBytes(stream, bytes);
+      decoder().flowController().writeWindowUpdates(ctx);
```

Comment:
This is no longer the case now that we have decided against batching window updates.